dist/assets/checkbox.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="black" width="24px" height="24px"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5V5h14v14zM17.99 9l-1.41-1.42-6.59 6.59-2.58-2.57-1.42 1.41 4 3.99z"/></svg>
```

Comment:
What do you think about using an empty checkbox icon that doesn't have a checkmark?

Right now, the checkmark has 2 different meanings depending on the icon (select or archive).

Reply:
I kinda wish we could have three states -- none selected, all selected, or some selected -- but I don't have much time to work on that change. I'll see later if I do.